interfaces/client.md

```diff
@@ -76,6 +76,15 @@ Empire will attempt to parse standard Mimikatz outputs and keep them in an inter
 
 ![credentials](https://user-images.githubusercontent.com/20302208/100279997-58c7a200-2f1c-11eb-9230-9becfb48bf9a.jpg)
 
+<<<<<<< Updated upstream
+=======
+Credentials can also be added manually. Use the `usecredentials add` command. There will be blank field with the field that need to be set. Set each field use `set`. Check that the fields are filled out with the `options` command. To save them into the credential menu save them with the `execute` command. 
+
+![manually-add-credentials](https://github.com/mwnickerson/empire-docs/blob/main/interfaces/images/manual-credentials.png)
```

Comment:
You can upload this image to GitHub by dragging it into an issue/pr comment to get a static url like: `https://user-images.githubusercontent.com/<name>.png`. This is preferred to pointing to your forked repo.